client/index.js

```diff
@@ -9,6 +9,9 @@ import App from './app';
 // establishes socket connection
 import './socket';
 
+// Where is the right place to do this?
```

Comment:
Maybe we write a button that the user can press to clear their local storage of their cart.